Scripts/CI/README_Metadata_StyleCheck/README_style_checker.py

```diff
@@ -0,0 +1,435 @@
+#!/usr/bin/env python3
+
+"""
+Comments in PyCharm style.
+References
+"""
+
+import os
+import re
+import typing
+import argparse
+
+# region Global sets
+# A set of words that get omitted during letter-case checks.
+exception_proper_nouns = {
+    'WmsLayer',
+    'ArcGIS Online',
+    'OAuth',
+    'Web Mercator',
+    'ArcGIS Pro',
+    'GeoPackage',
+    'loadStatus',
+    'Integrated Windows Authentication',
+    'GeoElement',
+    'Network Link',
+    'Network Link Control',
+    'Open Street Map',
+    'OpenStreetMap',
+    'Play KML Tour',
```

Comment:
Is this list for sample names only, specific sections of the readme, or any word inside a readme?
In case of the latter, we should add KML as well, but there's probably many more that we'd add along the way

Reply:
In hindsight, might be good to have this in a separate file as well, so that we have a central place to refer to the keywords

Reply:
This is specifically for the "tags" section and "title". It is for when it checks casing by comparing uppercase and lowercase so KML is ok as it will equal KML (but not kml) whereas e.g. GeoPackage will not equal GEOPACKAGE or geopackage. So in this list I've picked out all the titles that don't fit this rule (e.g. Play KML Tour) and any tags identified as not fitting this rule.

Other sections are left alone, e.g. Relevant API where they contain a lot / almost exclusively these pronouns.

Reply:
Ah now I follow! Thanks 😄